lib/config/plugins.js

```diff
@@ -105,18 +109,20 @@ module.exports = {
     load(pluginName) {
         const pluginNamespace = getNamespace(pluginName),
             pluginNameWithoutNamespace = removeNamespace(pluginName),
-            pluginNameWithoutPrefix = removePrefix(pluginNameWithoutNamespace);
+            pluginNameWithoutPrefix = removePrefix(pluginNameWithoutNamespace),
+            shortName = pluginNamespace + pluginNameWithoutPrefix,
+            longName = pluginNamespace + PLUGIN_NAME_PREFIX + pluginNameWithoutPrefix;
         let plugin = null;
 
-        if (!plugins[pluginNameWithoutPrefix]) {
+        if (!plugins[shortName]) {
             try {
-                plugin = require(pluginNamespace + PLUGIN_NAME_PREFIX + pluginNameWithoutPrefix);
+                plugin = require(longName);
             } catch (err) {
-                debug("Failed to load plugin eslint-plugin-" + pluginNameWithoutPrefix + ". Proceeding without it.");
+                debug("Failed to load plugin " + longName + ". Proceeding without it.");
```

Comment:
I'm okay with it, but you may want a second opinion. Thanks!

On Aug 19, 2016 9:32 AM, "Toru Nagashima" <notifications@github.com> wrote:

> In lib/config/plugins.js
> <https://github.com/eslint/eslint/pull/6939#discussion_r75491658>:
>
> >              } catch (err) {
> > -                debug("Failed to load plugin eslint-plugin-" + pluginNameWithoutPrefix + ". Proceeding without it.");
> > +                debug("Failed to load plugin " + longName + ". Proceeding without it.");
>
> I will fix it additionally in this PR.
> Is it OK that I simply remove Proceeding without it.?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/eslint/eslint/pull/6939/files/fe9d2f7e3dd14558c2df35d55dda4691ded1f888#r75491658>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AARWeqzbyDlDnr-QSvxMbISzBGwUW7-Mks5qhb5jgaJpZM4Jogs2>
> .
>